cms/src/hooks/sync-to-legacy/filters/lists.ts

```diff
@@ -0,0 +1,58 @@
+import { oldKnex } from "../oldKnex";
+import { createLocalizable, getStatusFromNew, isObjectUseless, upsertLS } from "../utils";
+import episodes from '../../../btv';
+import { LanguageEntity, CategoryEntity, SeriesEntity } from "@/Database";
+import { ItemsService } from "directus";
+
+export async function createList(p, m, c) {
+    console.log('Item created!');
+    console.log(p,m,c);
```

Comment:
In general I feel that there is a log of unfiltered logging. Maybe this should be cleaned up (and named correctly) to aid potential debugging.